rails/ordinals/gd.rb

```diff
@@ -0,0 +1,23 @@
+{
+  gd: {
+    number: {
+      nth: {
+        ordinals: -> (_key, number:, **_options) {
+          if number.to_i.abs == 1
+            'ᵈ'
+          elsif number.to_i.abs == 2
+            'ⁿᵃ'
+          elsif number.to_i.abs == 3
+            'ˢ'
+          else
+            'ᵐʰ'
+          end
```

Comment:
A minor change in syntax here can help reduce repetition and is slightly more performant:

```suggestion
          case number.to_i.abs
          when 1
            'ᵈ'
          when 2
            'ⁿᵃ'
          when 3
            'ˢ'
          else
            'ᵐʰ'
          end
```